README.rst

```diff
@@ -19,7 +19,7 @@ We release updates regularly, with a new version every 3 months. Find more detai
 * `Release notes`_
 * `Release process`_
 
-In 2020, we're working on improvements to the heart of pip. Please `learn more and take our survey`_ to help us do it right.
+In pip 20.3, we're making a big improvement to the heart of pip. Please `learn more`_ and `take our survey`_ to help us do it right.
```

Comment:
The [PSF blogpost link](https://pyfound.blogspot.com/2020/03/new-pip-resolver-to-roll-out-this-year.html) contains a link to the [sign-up form for the UX Studies](https://bit.ly/pip-ux-studies). It's not really a survey. So my suggestion would be:

> In pip 20.3, we're making a big improvement to the heart of pip. We want to your input, so `sign-up to take part in our UX research`_ to help us do it right.